### ISSUE TITLE
FulltextSearchable::enable() causes build error (3.3.1)

### ISSUE THREAD

**Issue by micschk** [silverstripe/silverstripe-framework#5126]
It seems the DB engine doesn't actually get updated to MyISAM, resulting in a "The used table type doesn't support FULLTEXT indexes" build-error when applying the FulltextSearchable extension in FulltextSearchable::enable().

Both MySQLDatabase and MySQLPDODatabase have this issue. 
Manually setting the engine to MyISAM seems to resolve it but this shouldn't be required as the enable method should update the tables to use the MyISAM table.

**Comment by tractorcow** [comment]
Ok, what happens if you do this.

```yaml
SiteTree:
  create_table_options:
    MySQLDatabase: 'ENGINE=MyISAM'
```

**Comment by tractorcow** [comment]
It doesn't work... bugfix inc.

**Comment by tractorcow** [comment]
Fix at https://github.com/silverstripe/silverstripe-framework/pull/5145

**Comment by sminnee** [comment]
Fixed in 3.2
